core/src/mindustry/world/blocks/experimental/BlockForge.java

```diff
@@ -18,6 +18,7 @@
 import mindustry.world.blocks.payloads.*;
 import mindustry.world.blocks.production.*;
 import mindustry.world.consumers.*;
+import mindustry.world.meta.*
```

Comment:
```suggestion
import mindustry.world.meta.*;
```

Reply:
why why why
I swear, the semicolon WAS THERE